<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranscriberWidget</class>
 <widget class="QWidget" name="TranscriberWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>135</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>135</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="EnableCheck">
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="DirectoryLabel">
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="7">
    <widget class="QLineEdit" name="DirectoryName"/>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="DirectoryButton">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="BrightnessLabel">
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Brightness</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="BirghtnessSpinBox">
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="NoteLabel">
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="NoteSpinBox">
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="EncodeLabel">
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Encode</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QComboBox" name="EncodeComboBox">
     <item>
      <property name="text">
       <string>Shift-JIS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EUC-JP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UTF-8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="ColorLable">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QComboBox" name="ColorSelector"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>DirectoryButton</sender>
   <signal>clicked()</signal>
   <receiver>TranscriberWidget</receiver>
   <slot>openDirDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BirghtnessSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TranscriberWidget</receiver>
   <slot>settingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NoteSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TranscriberWidget</receiver>
   <slot>settingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ColorSelector</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TranscriberWidget</receiver>
   <slot>settingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openDirDialog()</slot>
  <slot>settingChanged()</slot>
 </slots>
</ui>
